spec/features/create_new_work_spec.rb

```diff
@@ -121,7 +121,7 @@
 
           expect(page).to have_content('Keyword must be filled in')
 
-          fill_in 'Keyword', with: 'Springs'
+          fill_in 'keyword-input-field', with: 'Springs'
```

Comment:
Is there a reason we can't use the label here?  Using the label in the test helps to test that the form is accessible.

Reply:
once i added the ids to the elements, it stopped finding it by the label:

```
 Capybara::ElementNotFound:
       Unable to find field "Keyword" that is not disabled
```

but now that i moved this test to a request spec, we may not need the ids on the input elements anymore anyway...let me back that out

Reply:
I believe this was because the element already had an ID that was automatically generated. This id matched the label. When you added manual ids, it no longer aligned with the label.  Accessability was broken.